code/src/AlbumCover.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import { Heart} from 'icons/heart'
+import { Play} from 'icons/play'
+import { Dots} from 'icons/dots'
+import 'AlbumCover.css'
+
+export const AlbumCover = (props) => (
+  <div className="AlbumImage">
+    <div className="Icons">
+      <Heart />
+      <div className="Play">
+        <Play />
+      </div>
```

Comment:
You can put the className on the Play tag instead.
`<Play className="play" />`
also, don't start the classnames with capital letter.